core/domain/fs_services.py

```diff
@@ -183,24 +183,22 @@ def get(self, filepath: str) -> bytes:
     def commit(
         self,
         filepath: str,
-        raw_bytes: Union[bytes, str],
+        raw_bytes: bytes,
```

Comment:
Cna you please make sure that in all callsites of commit we always pass only `bytes`?

Reply:
@vojtechjelinek I have manually checked all the callsites of commit and ensured that `bytes` are only passed.